back/app/models/phase.rb

```diff
@@ -253,6 +264,19 @@ def pmethod
     @pmethod
   end
 
+  def set_manual_voters(amount, user)
+    return if amount == manual_voters_amount
+
+    self.manual_voters_amount = amount
+    self.manual_voters_last_updated_by = user if user
+    self.manual_voters_last_updated_at = Time.now
+  end
+
+  def update_manual_votes_count!
+    reload
```

Comment:
A bit weird to have the reload here, shouldn't the caller manage this responsibility?
If not, best to add a comment to explain the why